analytic/src/iosMain/kotlin/com/bunbeauty/analytic/AnalyticService.kt

```diff
@@ -0,0 +1,14 @@
+package com.bunbeauty.analytic
+
+import cocoapods.FirebaseAnalytics.FIRAnalytics
+import com.bunbeauty.analytic.event.FoodDeliveryEvent
+import com.bunbeauty.analytic.event.toMap
+import platform.Foundation.NSBundle
+
+actual class AnalyticService {
+    actual fun sendEvent(event: FoodDeliveryEvent){
+        val targetName = NSBundle.mainBundle.objectForInfoDictionaryKey("CFBundleName")
```

Comment:
It's copy/paste. Let's move `targetName` from `:shared` module to `:core` module and connect to `:analytic` and `:shared` modules

Reply:
module for 1 varible?